sizefashion/src/components/ProductImages.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { getImageTags } from '../utils/helpers';
+
+const ProductImages: React.FC<{ image: string | undefined }> = ({ image = '' }) => {
+  const imageSrc = getImageTags(image)
+
+  return (
+    <div>
```

Comment:
@otaaviostein we can safely remove this logicless `div` element and let the `img` element be the absolute source of this component.